tools/release_tool.py

```diff
@@ -193,13 +196,27 @@ def python_version(version):
 
 
 def sanity(venv, version, published=False):
-    version = python_version(version)
+    version = format_python_version(version)
     run_command("Creating venv", f"rm -rf {venv} && python -m venv {venv}")
     package = f"opendp=={version}" if published else f"python/wheelhouse/opendp-{version}-py3-none-any.whl"
-    run_command(f"Installing opendp {version}", f"source {venv}/bin/activate && pip install {package}")
+    run_command_with_retries(f"Installing opendp {version}", f"source {venv}/bin/activate && pip install {package}")
     run_command("Running test script", f"source {venv}/bin/activate && python python/example/test.py")
 
 
+def run_command_with_retries(description, args, retries = 3, wait_time_seconds = 5):
+  while retries > 0:
+    try:
+      return run_command(description, args)
+    except Exception as e:
+      was_final_attempt = retries == 1
+      if was_final_attempt:
+        log(f"{ANSI_FAIL}{e.stderr}{ANSI_END}")
+        raise e
+      log(f"Waiting {wait_time_seconds} Seconds | Retries Left: {retries - 1}")
+      time.sleep(wait_time_seconds)
+    retries -= 1
+
+
```

Comment:
Could you reformat to match our convention of 4 spaces indentation?

Also, for organization purposes, I'd suggest moving this function up to line 31 (immediately after `run_command`).

Thanks!